launchflow/gcp/firebase_site.py

```diff
@@ -0,0 +1,108 @@
+from dataclasses import dataclass
+from typing import List, Optional
+
+from launchflow import exceptions
+from launchflow.gcp.firebase import FirebaseHostingSite, FirebaseProject
+from launchflow.gcp.service import GCPStaticService
+from launchflow.models.enums import ServiceProduct
+from launchflow.node import Inputs
+from launchflow.resource import Resource
+from launchflow.service import DNSOutputs, DNSRecord, StaticServiceOutputs
+
+
+@dataclass
+class FirebaseStaticSiteInputs(Inputs):
+    pass
+
+
+class FirebaseStaticSite(GCPStaticService):
```

Comment:
nit: match the style of others with a link to the external documentation

Reply:
saving for follow up